_data/studies.tsv

```diff
@@ -61,37 +61,37 @@ idr0052-walther-condensinmap	experimentC	prod66	754	6	0	6		0	171	8720	0.00053110
-idr0085-walsh-mfhrem	experimentA	prod86	1202	7	0			0	7	15206	0.115773571855	115773571855	25	4630.9428742	2076 x 1681 x 1160 x 2 x 1
+idr0070-kerwin-hdbr	experimentA	prod84	1104	287	0			0	4554	13512	0.133538190509	133538190509	9232	14.464708677318024	8223 x 9270 x 1 x 3 x 1
+idr0073-schaadt-immuneinfiltrates	experimentA	prod80	1002	3	0			0	193	579	0.024520087958	24520087958	580	42.276013720689654	7905 x 6994 x 1 x 3 x 1
+idr0075-cabirol-honeybee	experimentA	prod73	951	10	0			0	20	1932	0.001014299685	1014299685	81	12.522218333333335	512 x 512 x 97 x 1 x 1
+idr0077-valuchova-flowerlightsheet	experimentA	prod84	1101	9	0			0	149	1005278	6.490732186705	6490732186705	1938	3349.1910148116617	1780 x 1780 x 99 x 4 x 32
+idr0079-hartmann-lateralline	experimentA	prod84	1102	20	0			0	350	69499	0.122001074424	122001074424	1708	71.4292004824356	1623 x 809 x 141 x 1 x 1
+idr0079-hartmann-lateralline	experimentB	prod84	1103	2	0			0	62	10917	0.01787342766	17873427660	312	57.28662711538462	1572 x 780 x 117 x 2 x 1
+idr0081-georgi-adenovirus	screenA	prod82	2401	2	768			2	768	1536	0.012925055458	12925055458	2310	5.595262103030303	2048 x 2048 x 1 x 2 x 1
+idr0081-georgi-adenovirus	screenB	prod82	2402	2	768			2	768	1536	0.012925618753	12925618753	2310	5.595505953679654	2048 x 2048 x 1 x 2 x 1
+idr0081-georgi-adenovirus	screenC	prod82	2403	2	768			2	768	1536	0.012908962639	12908962639	2310	5.588295514718615	2048 x 2048 x 1 x 2 x 1
+idr0081-georgi-adenovirus	screenD	prod82	2404	16	6144			16	6144	12288	0.103411092154	103411092154	18466	5.600080805480342	2048 x 2048 x 1 x 2 x 1
+idr0081-georgi-adenovirus	screenE	prod82	2405	16	6144			16	6144	12288	0.103262766389	103262766389	18464	5.592654158849654	2048 x 2048 x 1 x 2 x 1
+idr0083-lamers-sarscov2	experimentA	prod81	1051	1	0			0	2	2	0.071359237683	71359237683	7	10194.176811857144	111872 x 130304 x 1 x 1 x 1
+idr0084-oudelaar-alphaglobin	experimentA	prod85	1151	6	0			0	493	55258	0.115925645638	115925645638	1481	78.27525026198514	1024 x 1024 x 56 x 2 x 1
+idr0086-miron-micrographs	experimentA	prod85	1158	25	0			0	574	55454	0.029169557122	29169557122	1723	16.929516611723734	512 x 512 x 45 x 2 x 1
+idr0086-miron-micrographs	experimentB	prod85	1159	18	0			0	943	138555	0.072865285123	72865285123	2830	25.747450573498234	512 x 512 x 49 x 3 x 1
+idr0086-miron-micrographs	experimentC	prod85	1160	3	0			0	92	14484	0.007617483877	7617483877	277	27.499941794223826	512 x 512 x 52 x 3 x 1
+idr0086-miron-micrographs	experimentD	prod85	1161	2	0			0	11	10546	0.004104873039	4104873039	34	120.73155997058824	1018 x 602 x 503 x 2 x 1
+idr0087-paci-nuclearimport	experimentA	prod85	1157	38	0			0	456	50976	0.04848585023	48485850230	1370	35.391131554744526	640 x 640 x 1 x 3 x 37
+idr0048-abdeladim-chroms	experimentA	prod86	1201	1	0			0	2	4479	0.129647463006	129647463006	127	1020.8461654015748	11034 x 9271 x 747 x 3 x 1
+idr0085-walsh-mfhrem	experimentA	prod86	1202	3	0			0	7	15206	0.115773571855	115773571855	25	4630.9428742	2076 x 1681 x 1160 x 2 x 1
```

Comment:
I can see that these two lines were added at the end of the document, but should they not be rather inserted into the correct line to keep ascending order by the study number ?

Reply:
I don't think it is a requirement. My assumption is that this TSV file should extend as studies get released so the natural order will rather to have the `Introduced` column in ascending order.

Consumers of the TSV file like https://idr.openmicroscopy.org/about/studies.html should be able to do the filtering and sorting by their column of choice.